### PR TITLE
Video Intelligence 0.3.2

### PR DESCRIPTION
**Pull request by lukesneeringer**
This release fixes a bug with long-running operations.

Closes #2622.

### CODE REVIEW

File: packages/video-intelligence/src/index.js [hunk 1]
```diff
@@ -27,10 +21,12 @@ var gapic = {
   v1beta2: require('./v1beta2'),
 };
 var gaxGrpc = require('google-gax').grpc();
+var googleProtoFiles = require('google-proto-files');
```
Comment (landrito) [inline packages/video-intelligence/src/index.js]: I don't think this is needed in this file? Was this generated?

File: packages/video-intelligence/src/index.js [hunk 2]
```diff
@@ -13,12 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-/*!
```
Comment (stephenplusplus) [inline packages/video-intelligence/src/index.js]: Need!

File: packages/video-intelligence/src/index.js [hunk 3]
```diff
@@ -60,24 +56,23 @@ const VERSION = require('../package.json').version;
...
 
-var v1beta2Protos = {};
+var v1beta1Protos = {};
 
-extend(v1beta2Protos, gaxGrpc.loadProto(
-  path.join(__dirname, '..', 'protos',
-    'google/cloud/videointelligence/v1beta2/video_intelligence.proto')
-).google.cloud.videointelligence.v1beta2);
+extend(v1beta1Protos, gaxGrpc.loadProto(
+  path.join(__dirname, '..', 'protos'), 'google/cloud/videointelligence/v1beta1/video_intelligence.proto')
```
Comment (stephenplusplus) [inline packages/video-intelligence/src/index.js]: I think the linter's gonna have an issue with this one.

File: packages/video-intelligence/package.json [hunk 4]
```diff
@@ -26,10 +26,13 @@
   ],
   "dependencies": {
     "extend": "^3.0",
-    "google-gax": "^0.13.5"
+    "google-gax": "^0.14.0",
+    "google-proto-files": "^0.12.0",
```
Comment (stephenplusplus) [inline packages/video-intelligence/package.json]: I don't think this is actually used.

File: packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js [hunk 5]
```diff
@@ -76,11 +78,19 @@ function VideoIntelligenceServiceClient(gaxGrpc, loadedProtos, opts) {
     grpc: gaxGrpc.grpc
   }).operationsClient(opts);
 
+  var protoFilesRoot = new gax.grpc.GoogleProtoFilesRoot();
+  protoFilesRoot = protobuf.loadSync(
```
Comment (stephenplusplus) [inline packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js]: Is this right? The line before this assigns a value to `protoFileRoot`, then this overwrites it?

File: packages/video-intelligence/package.json [hunk 6]
```diff
@@ -26,10 +26,13 @@
...
+    "protobufjs": "^6.8.0"
   },
   "devDependencies": {
-    "mocha": "^3.2.0"
+    "mocha": "^3.2.0",
+    "through2": "^2.0.3"
```
Comment (stephenplusplus) [inline packages/video-intelligence/package.json]: Phantom dependency strikes again!

File: packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js (hunk 5 above)
Comment (landrito) [reply packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js]: Yep the root gets passed into the loader and uses that as a base for the protos that are loaded. The method _probably_ updates the passed in object but I'd rather not rely on that assumption.

File: packages/video-intelligence/package.json (hunk 6 above)
Comment (landrito) [reply packages/video-intelligence/package.json]: Blah, Noted. I'll add some logic around this in the generator.
https://github.com/googleapis/toolkit/issues/1585

File: packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js (hunk 5 above)
Comment (stephenplusplus) [reply packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js]: Ah, I missed that, sorry!

File: packages/video-intelligence/src/index.js (hunk 2 above)
Comment (lukesneeringer) [reply packages/video-intelligence/src/index.js]: @stephenplusplus Could you add a bug in googleapis/toolkit explaining what this is?

File: packages/video-intelligence/src/index.js (hunk 1 above)
Comment (lukesneeringer) [reply packages/video-intelligence/src/index.js]: Yes.

File: packages/video-intelligence/src/index.js (hunk 3 above)
Comment (lukesneeringer) [reply packages/video-intelligence/src/index.js]: I added a carriage return manually for now.

**Pull request merged**